cf/irsa-api-role.yaml

```diff
@@ -106,6 +106,7 @@ Resources:
                 Action: "s3:DeleteObject"
                 Resource:
                   - !Sub "arn:aws:s3:::biomage-originals-${Environment}/*"
+                  - !Sub "arn:aws:s3:::biomage-filtered-cells-${Environment}/*"
```

Comment:
The api doesn't need to interact with it except when a project/experiment is deleted, it needs to clean up both the `biomage-originals` and `biomage-filtered-cells` entries from their respective buckets